### DISCUSSION TITLE
Account reports including initial balance in odoo 17 accounting community edition??

### DISCUSSION THREAD

**Discussion by shyambista1** [OCA/odoo-community.org#179] [Q&A] [answered]
I'm looking for solutions or free modules for Odoo 17 Accounting Community Edition that include initial balances in their account reports. Specifically, I'm interested in finding modules that ensure initial balances are accurately reflected in reports like balance sheets or trial balances. If anyone has recommendations or has come across any community modules that handle this effectively, your input would be greatly appreciated. Thanks!

**Comment by pedrobaeza** [comment]
https://github.com/OCA/account-financial-reporting/tree/17.0/account_financial_report

**Comment by shyambista1** [comment]
Thank you for your response but that module does not provide reports like balance sheet, profit and loss and cash flow statement.

**Reply by pedrobaeza** [reply, marked as answer]
https://github.com/OCA/mis-builder/tree/17.0/mis_builder + migrate to 17 https://github.com/OCA/account-financial-reporting/tree/16.0/mis_template_financial_report or use specific country MIS reports.

**Comment by vdewulf** [comment]
Hello,
A Cash flow report is available here as well: https://github.com/OCA/account-financial-reporting/tree/16.0/mis_builder_cash_flow
In version 16 still, there is no pending pull request with a migration to v17 already created.
If you have technical skill or someone in your team does, creating a PR and contributing would be very helpful for the whole community!

For the profit and loss / balance, which country are you working in?

**Comment by shyambista1** [comment]
Thank you for the responses. Trial balance report works perfectly fine from [https://github.com/OCA/account-financial-reporting](https://github.com/OCA/account-financial-reporting/tree/16.0/mis_builder_cash_flow) but I was looking for balance sheet and profit and loss mainly.
I have tried some third party modules but those does not include the initial balance like how it does in the odoo enterprise reports. 
I am working in Nepal.

**Reply by vdewulf** [reply]
Hello,
Nepal! How nice! But I think you're one of the first to come to the OCA and look for community modules specifically for Nepal, because I don't even see any l10n-nepal in the list of all the OCA repository: https://github.com/orgs/OCA/repositories?language=&q=l10n&sort=&type=all&page=2
This could be a nice new addition to the OCA, if you are willing to author new reports for Nepal.

I personally started my contribution to the OCA as a functional analyst by creating and updating MIS Reports for Belgium, based on the MIS Bulder module which is very useful:
For example here: https://github.com/OCA/l10n-belgium/tree/16.0/l10n_be_mis_reports
As you can see in the data folder (https://github.com/OCA/l10n-belgium/tree/16.0/l10n_be_mis_reports/data) it's mainly a module with XML files, one XML file per report (we have a lot of different format of profit and loss (pl) and balance sheet (bs) in Belgium, that's why there are a lot of files). The module referenced by Pedro [here](https://github.com/OCA/account-financial-reporting/tree/16.0/mis_template_financial_report) is a template to create such reports.

Are you looking for a Community alternative to the Enterprise edition for yourself or your customers? Is the Enterprise report not satisfactory or is it a matter of open source philosophy?

If you consider that an alternative is desirable, I think that creating a new repo l10n-nepal, proposing MIS Report Profit and Loss / Balance Sheet dedicated to Nepal inside the OCA would be very nice. The easiest way to start this process is to get support from technical people, ideally from your team or yourself, being familiar with Git and Github process, and able to follow the contribution guidelines: 
https://odoo-community.org/resources/code
https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst

**Reply by rvalyi** [reply]
Hello, did you test this new module https://github.com/OCA/account-financial-reporting/pull/1115 ?

It doesn't seem to deal with initial balance, but it still claims to do Profit and Loss...

**Reply by pedrobaeza** [reply]
I insist that https://github.com/OCA/account-financial-reporting/tree/16.0/mis_template_financial_report  migrated to 17 (which is only change the version number basically) provides a generic P&L and balance sheet based on the account types.

**Comment by shyambista1** [comment]
Thanks all for the responses..
I have used this module for trial balance and it does include initial balance https://github.com/OCA/account-financial-reporting/tree/17.0/account_financial_report but used other third party packages for the profit and loss and balance sheet.
